misk/src/main/kotlin/misk/web/interceptors/MetricsInterceptor.kt

```diff
@@ -20,7 +20,11 @@ internal class MetricsInterceptor internal constructor(
     val (elapsedTime, result) = timed { chain.proceed(chain.httpCall) }
 
     val elapsedTimeMillis = elapsedTime.toMillis().toDouble()
-    val callingPrincipal = caller.get()?.principal ?: "unknown"
+    val callingPrincipal = when {
+      caller.get()?.service != null -> caller.get()?.service!!
+      caller.get()?.user != null -> "user"
```

Comment:
This should have some special characters in there to indicate it's "special" and not conflict with a service potentially named "user", hence why I suggested `<user>`

Reply:
👍🏻 will update